modules/ephemeral-storage-integration-between-shared-resources-insights-operator-and-openshift-builds.adoc

```diff
@@ -0,0 +1,15 @@
+:_mod-docs-content-type: REFERENCE
```

Comment:
IMO this is more of a concept module.
See https://redhat-documentation.github.io/modular-docs/#reference-module-definition 

https://redhat-documentation.github.io/modular-docs/#concept-module-definition